controllers/mattermost/clusterinstallation/controller.go

```diff
@@ -135,7 +137,8 @@ func (r *ClusterInstallationReconciler) Reconcile(request ctrl.Request) (ctrl.Re
 	if err != nil {
 		r.setStateReconcilingAndLogError(mattermost, reqLogger)
 		r.updateStatus(mattermost, status, reqLogger)
-		return reconcile.Result{RequeueAfter: time.Second * 3}, err
+		reqLogger.Error(err, "Error checking ClusterInstallation health")
```

Comment:
Do we even want to log an error here? Might be better to just log it as `info` since it's not an actionable error it's just that the reconciliation hasn't completed, right?

Reply:
I used `Error` to basically be aligned with what was logged before and technically the health check function can return an error when it, for example, cannot list some resources. Either way, I am open to change it - 0/5 on that.